tizen/src/constants.h

```diff
@@ -0,0 +1,83 @@
+#ifndef FLUTTER_PLUGIN_WORKMANAGER_CONSTANTS_H_
+#define FLUTTER_PLUGIN_WORKMANAGER_CONSTANTS_H_
+
+const char *kInitialize = "initialize";
+const char *kRegisterOneOffTask = "registerOneOffTask";
+const char *kRegisterPeriodicTask = "registerPeriodicTask";
+const char *kCancelTaskByUniqueName = "cancelTaskByUniqueName";
+const char *kCancelTaskByTag = "cancelTaskBytag";
+const char *kCancelAllTasks = "cancelAllTasks";
+const char *kUnknown = "unknown";
+
+const char *kIsInDebugModeKey = "isInDebugMode";
+const char *kCallhandlekey = "callbackHandle";
+
+const char *kUniquenameKey = "uniqueName";
+const char *kNameValueKey = "taskName";
+const char *kTagKey = "tag";
+const char *kExistingWorkpolicykey = "existingWorkPolicy";
+
+const char *kNetworkTypekey = "networkType";
+const char *kBatteryNotLowKey = "requiresBatteryNotLow";
+const char *kChargingKey = "requiresCharging";
+const char *kDeviceidlekey = "requiresDeviceIdle";
+const char *kStorageNotLowKey = "requiresStorageNotLow";
+
+const char *kInitialDelaySecondsKey = "initialDelaySeconds";
+const char *kbackOffPolicyTypeKey = "backoffPolicyType";
+const char *kBackOffPolicyDelayMillisKey = "backoffDelayInMilliseconds";
+const char *kOutofQuotaPolicyKey = "outOfQuotaPolicy";
+const char *kPayloadKey = "inputData";
+
+const char *kFrequencySecondsKey = "frequency";
+
+const char *kCancelTaskTagKey = "tag";
+const char *kCancelTaskUniqueNameKey = "uniqueName";
+
+const char *kForegroundChannelName =
+    "be.tramckrijte.workmanager/foreground_channel_work_manager";
+const char *kBackgroundChannelName =
+    "be.tramckrijte.workmanager/background_channel_work_manager";
+
+const char *kNotInitializedErrMsg =
+    "You have not properly initialized the Flutter WorkManager Package. \n"
+    "You should ensure you have called the 'initialize' function first! "
```

Comment:
Isn't it "You should ensure you have called the 'initialize' function first! `\n`"?
Do it need a space before '\n'?

Reply:
The original message don't have `\n` before `"Example : \n"` so I removed '\n' in first line.

Reply:
Can i check the original message?

Reply:
This is the message from original android plugin. [link](https://github.com/fluttercommunity/flutter_workmanager/blob/44ba91f2ab527688bd2c4fab270caaa2766ec707/android/src/main/kotlin/be/tramckrijte/workmanager/WorkmanagerCallHandler.kt#L77)